pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -169,7 +176,8 @@ const (
 	ServiceExists apis.ConditionType = "Service"
 	// DeploymentExists is the ConditionType set on the EventListener, which
 	// specifies Deployment existence.
-	DeploymentExists apis.ConditionType = "Deployment"
+	DeploymentExists     apis.ConditionType = "Deployment"
+	KnativeServiceExists apis.ConditionType = "Knative Service"
```

Comment:
Do we need this Status condition? Can we just show the Ready status from the Knative service?

Reply:
Yes i agree we can remove and show whatever is coming from Knative